### PR TITLE
OCPBUGS#13156: Updates to 4.11 dynamic plugin docs

### PR DESCRIPTION
**Pull request by opayne1**
OCPBUGS#13156: Updates to 4.11 dynamic plugin docs

Version(s):
4.11

Issue:
https://issues.redhat.com/browse/OCPBUGS-13156

Link to docs preview:
https://opayne1.github.io/oliviapreview/OCPBUGS-13156-2/web_console/dynamic-plugin/dynamic-plugin-overview.html
Release notes link update: https://opayne1.github.io/oliviapreview/OCPBUGS-13156-2/release_notes/ocp-4-11-release-notes.html#ocp-4-11-dynamic-plug-in-updates

QE review:
- [x] QE has approved this change.

Additional information:
Mostly file renaming and structural changes. Also implements feedback received from https://github.com/openshift/openshift-docs/pull/59868 another PR in this series of updates.

### CODE REVIEW

**Comment by opayne1** [comment]
@jhadvig and @yapei PTAL this should be the last of those updates to the dynamic plugin docs that rename to "plugin" and adds the other feedback discussed in the other PRs. Thanks!

**Review by jhadvig — APPROVED** [review]
/lgtm

File: web_console/dynamic-plugin/dynamic-plugins-reference.adoc [hunk 1]
```diff
@@ -0,0 +1,17 @@
+:_content-type: ASSEMBLY
+[id="dynamic-plugins-reference_{context}"]
+= Dynamic plugin reference
+include::_attributes/common-attributes.adoc[]
+:context: dynamic-plugins-reference
+
+toc::[]
+
+You can add extensions that allow you to customize your plugin. Those extensions are then loaded to the console at run-time.
```
Comment (kcarmichael08) [inline web_console/dynamic-plugin/dynamic-plugins-reference.adoc:9]:
```suggestion
You can add extensions that allow you to customize your plugin. Those extensions are then loaded to the console at runtime.
```

File: modules/dynamic-plugin-development.adoc [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+.Prerequisites
+* You must have an OpenShift cluster running.
+* You must have the OpenShift CLI (`oc`) installed.
+* You must have link:https://yarnpkg.com/[`yarn`] installed.
+* You must have link:https://www.docker.com/[Docker] v3.2.0 or newer or link:https://podman.io/[Podman] installed and running.
+
+.Procedure
+
+. In your terminal, run the following command to install the dependencies for your plugin using yarn.
```
Comment (JoeAldinger) [inline modules/dynamic-plugin-development.adoc:19]:
```suggestion
. In your terminal, run the following command to install the dependencies for your plugin using Yarn.
```

File: modules/dynamic-plugin-development.adoc [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
++
+[source,terminal]
+----
+$ yarn install
+----
+
+. After installing, run the following command to start yarn.
```
Comment (JoeAldinger) [inline modules/dynamic-plugin-development.adoc:27]:
```suggestion
. After installing, run the following command to start Yarn.
```

File: modules/dynamic-plugin-development.adoc [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+. Run the {product-title} web console in a container connected to the cluster you have logged into by running the following command:
++
+[source,terminal]
+----
+$ yarn run start-console
+----
+
+.Verification
+* Visit link:http://localhost:9000/example[localhost:9000] to view the running plugin. Inspect the value of `window.SERVER_FLAGS.consolePlugins` to see the list of plugins which load at runtime.
```
Comment (JoeAldinger) [inline modules/dynamic-plugin-development.adoc:50]:
```suggestion
* Visit link:http://localhost:9000/example[localhost:9000] to view the running plugin. Inspect the value of `window.SERVER_FLAGS.consolePlugins` to see the list of plugins that load at runtime.
```

**Review by JoeAldinger — COMMENTED** [review]
A few suggestions. I'll be so happy to see this PR finished and merged 😄

File: modules/adding-tab-pods-page.adoc [hunk 5]
```diff
@@ -56,5 +102,18 @@ export default function ExampleTab() {
 }
 ----
 
+. Install a Helm chart with the name of the plugin as the Helm release name into a new namespace or an existing namespace as specified by the `-n` command-line option to deploy your plugin on a cluster. Provide the location of the image within the `plugin.image` parameter by using the following command:
```
[the remark this answers is not in the document]
Comment (opayne1) [reply modules/adding-tab-pods-page.adoc:105]: Revised to 

> Install a Helm chart with the name of the plugin as the Helm release name into a new namespace or an existing namespace as specified by the `-n` command-line option and provide the location of the image within the `plugin.image` parameter to deploy your plugin on a cluster by using the following command

**Comment by yapei** [comment]
Looks good to me

**Pull request merged**